includes/services/OpenLayers/jquery.openlayers.js

```diff
@@ -100,6 +100,7 @@
 				markerData.lonlat.transform(new OpenLayers.Projection("EPSG:4326"), new OpenLayers.Projection("EPSG:900913"));
 			}
 
+			if (this.bounds != null) this.bounds.extend(markerData.lonlat); // Extend the bounds when no center is set.
```

Comment:
The way I'm reading this code is that the bounds will get extended every time a marker is added. Is that incorrect?